docs/standard/modern-web-apps-azure-architecture/azure-hosting-recommendations-for-asp-net-web-apps.md

```diff
@@ -3,10 +3,10 @@ title: Azure hosting recommendations for ASP.NET Core web apps
 description: Architect Modern Web Applications with ASP.NET Core and Azure | Azure hosting recommendations for ASP.NET web apps
 author: ardalis
 ms.author: wiwagn
-ms.date: 10/07/2017
+ms.date: 06/11/2018
 ---
- 
-# Azure Hosting Recommendations for ASP.NET Core Web Apps
+
+# Azure hosting recommendations for ASP.NET Core web apps
 
 > "Line-of-business leaders everywhere are bypassing IT departments to get applications from the cloud (aka SaaS) and paying for them like they would a magazine subscription. And when the service is no longer required, they can cancel the subscription with no equipment left unused in the corner."  
```

Comment:
the first paragraph has a huge weight for SEO purposes. I'm wondering if these quotes as the 1st thing on each article should be moved lower in the article to allow the first paragraph to truly be what's going to be covered here. I also think that all the summaries can be further improved to explain what customers will find on each page without having to scan the whole thing to find out.